web_app/templates/finder.html

```diff
@@ -3,32 +3,21 @@
 
 {% block content %}
 
-    <!DOCTYPE html>
-    <html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
-        <title>Search Results</title>
-    </head>
-    <body>
-    
     <div class="container mt-5">
         <h1 class="text-center">Search Results</h1>
+
+        {{ market_list }}
```

Comment:
when the router passes a variable to the page, we need to access it inside the page using double curlies, like this.

your next step will be to use a jinja loop through each item in the list instead (see example shared in Slack)